__init__.py

```diff
@@ -457,14 +457,14 @@ def on_gui_page_show(self, message):
 
             # If a skill overrides the idle do not switch page
             override_idle = message.data.get('__idle')
-            if override_idle is True:
+            if override_idle:
                 # Disable idle screen
                 self.idle_override_set_time = time.monotonic()
                 self.log.info('Cancelling Idle screen')
                 self.cancel_idle_event()
                 self.override_idle = (message, time.monotonic())
 
-            elif isinstance(override_idle, int) and override_idle is not False:
+            elif isinstance(override_idle, int) and not override_idle:
```

Comment:
This handles a bit of a python issue

False is derived from int so this is an explicit case that needs to be tackled.

`>>> isinstance(False, int)`

Reply:
I think it may be clearest / most clear if we make a helper

```python
def is_override_time(override):
    """Check that the 
    return isinstance(override_idle, int) and not isinstance(override_idle, bool)
```

Then make the comparison

```python
        elif is_override_time(override_idle) or override_idle:
```

(could have `override_idle is True` if we wish to be really explicit)